dhcpv4/option_domain_search.go

```diff
@@ -0,0 +1,61 @@
+package dhcpv4
+
+// This module defines the OptDomainSearch structure.
+// https://tools.ietf.org/html/rfc3397
```

Comment:
use the text link for consistency with other options, https://tools.ietf.org/rfc/rfc3397.txt

Reply:
I think all options are using https://tools.ietf.org/html/rfc2132